src/models/charge/Charge.class.ts

```diff
@@ -56,9 +32,9 @@ class Charge {
   readonly returnUrl: string
   readonly paymentProvider: string
   readonly createdDate: string
-  readonly refundSummary?: RefundSummary // todo use common models, shared with tx
-  readonly settlementSummary?: SettlementSummary // todo use common models, shared with tx
-  readonly authorisationSummary?: AuthorisationSummary // todo use common models, shared with tx
+  readonly refundSummary?: RefundSummary // TODO: refactor to use common model once data structures are aligned
```

Comment:
this won't happen as Ledger includes the `amount_refunded` property in the refund summary, whereas Connector doesn't. You could have a `RefundSummary` class which doesn't include it, and a `LedgerRefundSummary` which extends that, and includes the `amountRefunded` field. This seems preferable over making it an optional paramter on the refund summary object, since we know it will be present for Ledger transactions, but not for Connector charges